docs/translate-client.rst

```diff
@@ -0,0 +1,15 @@
+Translate Client
+================
+
+.. automodule:: gcloud.translate.client
+  :members:
+  :undoc-members:
```

Comment:
Not a blocker for this PR, but I think we should remove the `:undoc-members:` bits everywhere (and fix whatever seems wrong, or breaks under docs coverage).